### ISSUE TITLE
Attacks won't base their level off of defaulted skills.

### ISSUE THREAD

**Issue by Angular-Angel** [richardwilkes/gcs#76]
So, for example, if I have an innate attack skill at default and a talent that boosts it, my innate attack won't count that skill and will instead base itself off of DX, even when that would be lower.

**Comment by Angular-Angel** [comment]
I've got the project compiling on my machine, so I figured I'd take a crack at this. Any idea where I should look? I would guess either weapons, or the look up code for skills? :/

**Comment by richardwilkes** [comment]
It's likely in the skill lookup... probably intentionally omitting zero-point skills for some other reason that would need to be dealt with.

**Comment by AndyDeck** [comment]
I found a (partial) workaround for this issue when playing with a mage's Missile spells.  I found that by adding the default's rule to the spell's Weapon skill details I could see the correct skill level in the appropriate Weapons block of the character sheet.  For instance, this character only spent points on Innate Attack (Projectile), achieving a level of 18 in that skill.  Innate Attack (Beam) therefore defaults to 16.  The Lightning spell has a Ranged Weapon entry that uses IA(Beam), and originally only listed DX-4 and IA(Beam)+0 as skills. I added a line for IA-2 with no specialization at the bottom, and the Lightning entry under my Ranged Weapons now correctly shows 16.
I have a side question that I think I'll file as a separate bug.
